tests/zero_shot_classifier_processor_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for ZeroShotClassifier processor.
+"""
+from typing import Dict
```

Comment:
To run the unit tests, we now set up the dependency of each module separately in the workflow. You can take a look at here to see if you can set up the test in the same way:

Reply:
Sounds good. Could you share the link that I could follow?

Reply:
https://github.com/asyml/forte-wrappers/blob/main/.github/workflows/main.yml#L76